Cargo.toml

```diff
@@ -22,14 +22,14 @@ appveyor = { repository = "rust-random/rand" }
 [features]
 default = ["std"] # without "std" rand uses libcore
 nightly = ["simd_support"] # enables all features requiring nightly rust
-std = ["rand_core/std", "alloc", "rand_os", "rand_jitter/std"]
+std = ["rand_core/std", "alloc", "getrandom"]
 alloc = ["rand_core/alloc"]  # enables Vec and Box support (without std)
 i128_support = [] # enables i128 and u128 support
 simd_support = ["packed_simd"] # enables SIMD support
 serde1 = ["rand_core/serde1", "rand_isaac/serde1", "rand_xorshift/serde1"] # enables serialization for PRNGs
 # re-export optional WASM dependencies to avoid breakage:
-wasm-bindgen = ["rand_os/wasm-bindgen"]
-stdweb = ["rand_os/stdweb"]
+wasm-bindgen = ["getrandom/wasm-bindgen"]
+stdweb = ["getrandom/stdweb"]
```

Comment:
I am not sure if we need to bubble these features. Users can write in application crates `getrandom = { version = "*", features = [ .. ] }`. Something similar will have to be done if `rand` is used indirectly via dependencies, which I think will be fairly common.

Reply:
We do if we don't want to make a breaking change — though we can't ever lose these without a breaking change.

Reply:
Well, v0.7 is already a breaking release, so we could use this chance to cleanup those features. Though if you want to keep number of breaking changes to a minimum, then it's fine to keep it as-is.

Reply:
On second thoughts, no, lets keep the feature forwarding. It's convenient for use in our tests and something we might have some other reason to break in the future (hopefully eventually these features won't be needed), so there's not much sense in a breaking change to achieve so little.